soccer/gameplay/plays/restarts/basic_indirect.py

```diff
@@ -0,0 +1,53 @@
+import standard_play
+import behavior
+import skills.move
+import skills.pivot_kick
+import constants
+import robocup
+import math
+import main
+import tactics.coordinated_pass
+import evaluation.passing_positioning
+import enum
+
+
+class BasicIndirect(standard_play.StandardPlay):
+
+    def __init__(self, indirect=None):
+        super().__init__(continuous=True)
+
+        self.add_transition(behavior.Behavior.State.start,
+                            behavior.Behavior.State.running, lambda: True,
+                            'immediately')
+
+        # FIXME: this could also be a PivotKick
+        kicker = skills.line_kick.LineKick()
```

Comment:
I think for any restarts, we've been doing line kick so that we don't have to worry about double touch stuff and it's a little more consistent I think